tests/test_pipeline.py

```diff
@@ -23,6 +24,22 @@ def test_pipeline(self):
         self.assertIsInstance(pipeline_result.model, Model)
         self.assertIsInstance(pipeline_result.model.regularizer, NoRegularizer)
 
+        model = pipeline_result.model
+        nations = Nations()
+        testing_mapped_triples = nations.testing.mapped_triples.to(model.device)
+
+        tails_df = model.predict_tails('brazil', 'intergovorgs', testing=testing_mapped_triples)
```

Comment:
@cthoyt We can additionally check for the df format (e.g. columns, num_rows). This will help us to detect "API-violating" changes, and some users might rely on the dataframe format for further processing.

Reply:
Done in b91425b